src/lib/admin/admin-pii.ts

```diff
@@ -0,0 +1,29 @@
+import { EncryptionService } from "../services/encryption";
+
+/**
+ * Structure of the data that goes into Admin::adminEncryptedPii
+ */
+export type AdminPii = {
+  adminName: string;
+  adminEmail: string;
+  adminPhoneNumber: string;
+};
+
+export async function encryptAdminPii(
+  pii: AdminPii,
+  encryptionService: EncryptionService,
+): Promise<string> {
+  const data = JSON.stringify(pii);
```

Comment:
Use JSON schema to serialise PII so that change to the PII schema can be managed better.

Reply:
Deferred to another ticket.